node/src/components/network/message.rs

```diff
@@ -0,0 +1,12 @@
+use std::fmt::{self, Debug, Display, Formatter};
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
```

Comment:
We need to look at messages and consider if we are moving to a more generic scheme of publishing objects (e.g. via CEP21). If that's the case, maybe we can finally get rid of any "custom" message and just use the equivalent of what is now `GetRequest`.

Reply:
If we do that, I'd prefer it be done once/if CEP-0021 is finalised and agreed.